### DISCUSSION TITLE
dsc.powerChanged

### DISCUSSION THREAD

**Discussion by jimtng** [taligentx/dscKeybusInterface#104] [dscKeybusInterface Discussion]
Under what condition would dsc.powerChanged be set? I tried unplugging the power supply to the panel, I only got dsc.troubleChanged. It didn't set dsc.powerChanged.

**Comment by taligentx** [comment]
The panel has a configurable delay before sending this status because it can get reported to a monitoring company, details towards the bottom of the README.md.

**Comment by taligentx** [comment]
Closing, if changing the panel configuration doesn't resolve this, please re-open.
